app/views/custom_fields/_form.html.erb

```diff
@@ -69,9 +69,17 @@ See COPYRIGHT and LICENSE files for more details.
           <span class="form--field-container">
             <span class="form--check-box-container">
               <input disabled="disabled" class="-cf-ignore-disabled form--check-box" type="checkbox" name="custom_field_multi_value_disabled">
-              <%= render partial: 'enterprise/locked_note' %>
             </span>
           </span>
+          <div class="form--field-instructions -no-italic -xwide">
+            <%=
+              angular_component_tag 'op-enterprise-banner',
```

Comment:
minor: indention here is different from the other `op-enterprise-banner` above (e.g. in `app/views/admin/settings/work_packages_settings/_enterprise_feature_hint.html.erb`). What is the correct indention?

Reply:
I don't know which one is "correct". Since rubocop did not complain, I assume both.. However, since all other places have the opening tag and the component name in one line, I will adapt this accordingly. 👍